src/java.base/share/classes/javax/crypto/AEADBadTagException.java

```diff
@@ -38,14 +38,14 @@ public class AEADBadTagException extends BadPaddingException {
     private static final long serialVersionUID = -488059093241685509L;
 
     /**
-     * Constructs a AEADBadTagException with no detail message.
+     * Constructs an <code>AEADBadTagException</code> with no detail message.
```

Comment:
Why not the {@code } as in other files?

Reply:
I started with java.crypto first, and the first few files I looked at had '<code>' so I used '<code>'. I later realized there was a mix of styles. I tried to continue with whatever style a file was using.

Reply:
using `{@code }`